src/extension/utils.js

```diff
@@ -542,32 +542,32 @@ export async function getProjectEnv({
       credentials: 'include',
       headers: authToken ? { 'x-auth-token': authToken } : {},
     };
-    res = await fetch(`https://admin.hlx.page/sidekick/${owner}/${repo}/${ref}/env.json`, options);
+    res = await fetch(`https://admin.hlx.page/sidekick/${owner}/${repo}/${ref}/config.json`, options);
   } catch (e) {
     log.warn(`unable to retrieve project config: ${e}`);
   }
   if (res && res.ok) {
     const {
-      preview,
-      live,
-      prod,
+      previewHost,
+      liveHost,
+      host,
       project,
-      contentSourceUrl,
+      mountpoints,
     } = await res.json();
-    if (preview && preview.host) {
-      env.previewHost = preview.host;
+    if (previewHost) {
+      env.previewHost = previewHost;
     }
-    if (live && live.host) {
-      env.liveHost = live.host;
+    if (liveHost) {
+      env.liveHost = liveHost;
     }
-    if (prod && prod.host) {
-      env.host = prod.host;
+    if (host) {
+      env.host = host;
     }
     if (project) {
       env.project = project;
     }
-    if (contentSourceUrl) {
-      env.mountpoints = [contentSourceUrl];
+    if (mountpoints) {
+      env.mountpoints = Array.isArray(mountpoints) ? mountpoints : [mountpoints];
```

Comment:
Can it ever not be an array?

Reply:
As discussed, I have removed the explicit check for array type in [1b6af8c](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/1b6af8c69c6e2fee15ed79f5a97d456f5b96c23c)(trusting that admin service will always return an array)